### ISSUE TITLE
Accessibility issues in ix-button

### ISSUE THREAD

**Issue by krasnoff** [siemens/ix#2737]
### Prerequisites

- [x] I have read the [Contributing Guidelines](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md).
- [x] I have not leaked any internal/restricted information like screenshots, videos, code snippets, links etc.

### What happened?

The `ix-button` component in **disabled** mode does not meet the accessibility standards regarding color contrast between forground and background according to **WCAG AA**

please see examples in the following pages:
https://ix.siemens.io/demo/v2/preview/html/preview-examples/buttons.html?no-margin=true&theme=brand&colorSchema=dark

https://ix.siemens.io/demo/v2/preview/html/preview-examples/buttons.html?no-margin=true&theme=classic&colorSchema=dark

https://ix.siemens.io/demo/v2/preview/html/preview-examples/buttons.html?no-margin=true&theme=classic&colorSchema=light

https://ix.siemens.io/demo/v2/preview/html/preview-examples/buttons.html?no-margin=true&theme=brand&colorSchema=light

### What type of frontend framework are you seeing the problem on?

JavaScript, Others, React, Angular (Standalone), Angular (NgModule)

### On which version of the frontend framework are you experiencing the issue?

its a css issue, not a front end framework issue

### Which version of iX do you use?

v5.0.0

**Comment by flxlst09** [comment]
Hi @krasnoff Thanks for reaching out. We also discussed the contrast rations for disabled elements earlier, however the WCAG states an exception for inactive elements:

"User Interface Components that are not available for user interaction (e.g., a disabled control in HTML) are not required to meet contrast requirements. An inactive user interface component is visible but not currently operable. An example would be a submit button at the bottom of a form that is visible but cannot be activated until all the required fields in the form are completed."
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html
